app/qml/components/MMComboBoxDrawer.qml

```diff
@@ -0,0 +1,198 @@
+ *   (at your option) any later version.                                   *
+ *                                                                         *
+ ***************************************************************************/
+
+import QtQuick
+import QtQuick.Controls
+import QtQuick.Controls.Basic
+import "../inputs"
+
+Drawer {
+  id: control
+
+  property alias title: title.text
+  property alias model: listView.model
+  property alias dropDownTitle: title.text
+  property bool multiSelect: false
+
+  padding: 20 * __dp
+
+  signal featureClicked( var selectedFeatures )
+
+  width: ApplicationWindow.window.width
+  height: (mainColumn.height > ApplicationWindow.window.height ? ApplicationWindow.window.height : mainColumn.height) - 20 * __dp
+  edge: Qt.BottomEdge
+
+  Rectangle {
+    color: roundedRect.color
+    anchors.top: parent.top
+    anchors.left: parent.left
+    anchors.right: parent.right
+    height: 2 * radius
+    anchors.topMargin: -radius
+    radius: 20 * __dp
+  }
+
+  Rectangle {
+    id: roundedRect
+
+    anchors.fill: parent
+    color: __style.whiteColor
+
+    Column {
+      id: mainColumn
+
+      width: parent.width
+      spacing: 20 * __dp
+      leftPadding: control.padding
+      rightPadding: control.padding
+      bottomPadding: control.padding
+
+      Row {
+        width: parent.width - 2 * control.padding
+        anchors.horizontalCenter: parent.horizontalCenter
+
+        Item { width: closeButton.width; height: 1 }
+
+        Text {
+          id: title
+
+          anchors.verticalCenter: parent.verticalCenter
+          font: __style.t2
+          width: parent.width - closeButton.width * 2
+          color: __style.forestColor
+          horizontalAlignment: Text.AlignHCenter
+          verticalAlignment: Text.AlignVCenter
+          elide: Text.ElideRight
+        }
+
+        Image {
+          id: closeButton
+
+          source: __style.closeButtonIcon
+
+          MouseArea {
+            anchors.fill: parent
+            onClicked: control.visible = false
+          }
+        }
+      }
+
+      MMSearchEditor {
+        id: searchBar
+
+        width: parent.width - 2 * control.padding
+        placeholderText: qsTr("Text value")
+        bgColor: __style.lightGreenColor
+        visible: control.model.count > 4
+
+        onSearchTextChanged: function(text) {
+          control.model.searchExpression = text
+        }
+      }
+
+      ListView {
+        id: listView
+
+        bottomMargin: primaryButton.visible ? primaryButton.height + 20 * __dp : 0
+        width: parent.width - 2 * control.padding
+        height: {
+          if(control.model.count > 4) {
+            if(ApplicationWindow.window)
+              return ApplicationWindow.window.height - searchBar.height - 100 * __dp
+            else return 0
+          }
+          if(control.model)
+            return control.model.count * __style.menuDrawerHeight //+ primaryButton.height
```

Comment:
Commented out code

Reply:
Let's rather define a different variable than reusing one that is not really related to this context (`menuDrawerHeight `)